pkg/tfgen/generate_csharp.go

```diff
@@ -64,7 +64,7 @@ type declarer interface {
 type csharpNestedType struct {
 	declarer declarer
 	isInput  bool
-	isPlain  bool
+	isInvoke bool
```

Comment:
Was this change necessary? `isPlain` was intended to indicate that the type did not need to be marked up with `[Output]` attributes; it seems like it could serve the same purpose. To that end, I'd love to see a diff for the `Config.cs` files with these changes.

Reply:
I did not find any usage of `isPlain` other than passing it around. What about `Config.cs` that had to change? My test run on Azure doesn't yield any.